.github/workflows/release_js_sdk.yml

```diff
@@ -35,9 +38,14 @@ jobs:
       env:
         NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
 
+    - name: Update version
+      run: npm version ${{ github.event.inputs.version }} --no-git-tag-version
+
+    - name: Commit and push
+      run: git commit -am "Release ${{ github.event.inputs.version }}" && git push
+
     - name: pnpm build
       run: pnpm build
       
     - name: pnpm publish
       run: pnpm publish --no-git-checks
```

Comment:
Consider using the isRCRelease input to modify the npm publish command for RC releases:
```yaml
- name: pnpm publish
  run: pnpm publish --no-git-checks ${{ github.event.inputs.isRCRelease && '--tag rc' || '' }}
```